.eslintrc.yml

```diff
@@ -126,7 +126,7 @@ rules:
   no-self-compare: warn
   no-sequences: warn
   no-shadow-restricted-names: warn
-  no-shadow: warn
+  # no-shadow: warn
```

Comment:
My only concern is this won't survive a copier update

Reply:
Then maybe the way to go would be to have an special eslint in copier for version 14-15 (something like) https://github.com/OCA/oca-addons-repo-template/blob/master/src/%7B%25%20if%20odoo_version%20%3E%2012%20%25%7D.eslintrc.yml%7B%25%20endif%20%25%7D